File: app/src/main/java/com/github/brianspace/moviebrowser/di/modules/FavoriteMovieListFragmentModule.java
package com.github.brianspace.moviebrowser.di.modules;

import com.github.brianspace.moviebrowser.ui.fragment.FavoriteMovieListFragment;
import dagger.Module;
import dagger.android.ContributesAndroidInjector;

/**
 * Dagger module for MainActivity.
 */
@Module
public abstract class FavoriteMovieListFragmentModule {
    @ContributesAndroidInjector
    /* default */ abstract FavoriteMovieListFragment contributeFavoriteMovieListFragmentInjector();
}

File: app/src/main/java/com/github/brianspace/moviebrowser/di/modules/ui/FavoriteMovieListFragmentModule.kt
package com.github.brianspace.moviebrowser.di.modules.ui

import com.github.brianspace.moviebrowser.ui.fragment.FavoriteMovieListFragment
import dagger.Module
import dagger.android.ContributesAndroidInjector

/**
 * Dagger module for MainActivity.
 */
@Module
abstract class FavoriteMovieListFragmentModule {
    @ContributesAndroidInjector
    internal abstract fun contributeFavoriteMovieListFragmentInjector(): FavoriteMovieListFragment
}
